contrib/depends/toolchain.cmake.in

```diff
@@ -2,8 +2,8 @@
 SET(CMAKE_SYSTEM_NAME @depends@)
 SET(CMAKE_BUILD_TYPE release)
 
-SET(STATIC true)
-SET(UNBOUND_STATIC true)
+SET(STATIC ON)
+SET(UNBOUND_STATIC ON)
```

Comment:
Are those actually fixing something ?

Reply:
No, just makes it coherent finally.

Reply:
I'd rather not then. The existing is correct.

Reply:
So just keep as it is even though it is incoherent?

Reply:
Actually, what do you mean by incoherent ?

Reply:
+1 to @moneromooo-monero; I consider this type of change as noise.